imports/plugins/core/system-info/register.js

```diff
@@ -0,0 +1,17 @@
+import Reaction from "/imports/plugins/core/core/server/Reaction";
+import queries from "./server/no-meteor/queries";
+import resolvers from "./server/no-meteor/resolvers";
+import schemas from "./server/no-meteor/schemas";
+
+Reaction.registerPackage({
+  label: "System Information",
+  name: "reaction-systeminfo",
+  icon: "fa fa-info",
+  autoEnable: true,
+  version: "1.0.0",
```

Comment:
So far we have not been versioning core plugins separately. `version` field is optional and can be removed.

Reply:
oops added for testing and never removed it